webpack.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
                 test: /\.scss$/,
                 loaders: ["css-loader", "sass-loader"],
             },
+            {
+                test: /\.css$/,
```

Comment:
Hmm, won't this duplicate quite a bit?

Reply:
This can be removed for now. Importing CSS files isn't working yet.